src/main/java/uk/gov/hmcts/cmc/claimstore/services/staff/content/ClaimContentProvider.java

```diff
@@ -33,34 +34,44 @@ public ClaimContent createContent(Claim claim) {
         requireNonNull(claim);
 
         List<BigDecimal> totalAmountComponents = new ArrayList<>();
-        totalAmountComponents.add(((AmountBreakDown)claim.getClaimData().getAmount()).getTotalAmount());
+        totalAmountComponents.add(((AmountBreakDown) claim.getClaimData().getAmount()).getTotalAmount());
         totalAmountComponents.add(claim.getClaimData().getPayment().getAmountInPounds());
 
-
         InterestContent interestContent = null;
         if (!claim.getClaimData().getInterest().getType().equals(Interest.InterestType.NO_INTEREST)) {
             interestContent = interestContentProvider.createContent(
                 claim.getClaimData().getInterest(),
                 claim.getClaimData().getInterestDate(),
-                ((AmountBreakDown)claim.getClaimData().getAmount()).getTotalAmount(),
+                ((AmountBreakDown) claim.getClaimData().getAmount()).getTotalAmount(),
                 claim.getCreatedAt()
             );
             totalAmountComponents.add(interestContent.getAmountUpToNowRealValue());
         }
 
+        String signerName = null;
+        String signerRole = null;
+
+        if (claim.getClaimData().getStatementOfTruth().isPresent()) {
+            StatementOfTruth statementOfTruth = claim.getClaimData().getStatementOfTruth().get();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Call "Optional#isPresent()" before accessing the value. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.reform.hmcts.net/coding_rules#rule_key=squid%3AS3655)